core/src/main/resources/application.conf

```diff
@@ -23,6 +23,12 @@ configManager {
 
 ktor {
   deployment {
+    publicScheme = "http"
+    publicScheme = ${?SCHEME}
+    publicFqdn = "localhost"
+    publicFqdn = ${?FQDN}
```

Comment:
Calling the variables `PUBLIC_SCHEME` and `PUBLIC_FQDN` would make it more obvious that they belong together with `PUBLIC_PORT`.